products/android_studio.md

```diff
@@ -0,0 +1,103 @@
+The Android Studio IDE is built on top of the IntelliJ Platform.
+This relationship means that Android Studio plugins can be created using IntelliJ IDEA, Java or Kotlin, and the IntelliJ Platform.
+Android Studio plugins can also be packaged and distributed at [plugins.jetbrains.com](https://plugins.jetbrains.com), like plugins for JetBrains IntelliJ Platform-based products.
+
+Android Studio plugins extend or add functionality to the Android Studio IDE.
+They are not Android modules or apps to be run in the Android operating system such as on a smartphone or tablet.
+
+## Configuring IntelliJ Platform Projects for Android Studio Plugin Development
+An Android Studio plugin project is an example of cross-development: plugins projects are created in IntelliJ IDEA but intended to be run in a different IntelliJ Platform-based IDE.
+The configuration for an Android Studio plugin project is similar to the configuration for an IntelliJ IDEA plugin because both IDEs are based on the IntelliJ Platform.
+
+To create a new Android Studio plugin project, follow the tutorial on the [Getting Started with Gradle](/tutorials/build_system/prerequisites.md) page.
+The tutorial will produce a skeleton project suitable to use as a starting point for an Android Studio plugin.
+Note that on the [New Project Configuration Screen](/tutorials/build_system/prerequisites.md#new-project-configuration-screen) of the New Project Wizard tutorial: 
+* Choose Gradle from the product category pane as described in the tutorial, **not** _Android_.
+* Choose either Java and/or Kotlin/JVM as desired in the _Additional Libraries and Frameworks_ pane, but the tutorial continues assuming Java.
+  See the SDK code sample `kotlin_demo` for an example of a generic IntelliJ IDEA plugin based on Kotlin.
+  
+Some minor modifications to the skeleton project are needed, as discussed below.
+
+### Matching Versions of the IntelliJ Platform with the Android Studio Version
+For API compatibility it is important to match the version of the IntelliJ Platform APIs used for plugin development with the target version of Android Studio.
+The version number of Android Studio contains the version of the underlying IntelliJ Platform APIs that were used to build it.
+
+To find the version of the IntelliJ Platform used to build Android Studio, see the example _About_ dialog screen below.
+In this case the (BRANCH.BUILD.FIX) version of the IntelliJ Platform is `191.8026.42`, which corresponds to the IntelliJ IDEA version 2019.1.4.
+The [`build.gradle` configuration steps](#configuring-the-plugin-buildgradle-file) below explain how to set the IntelliJ Platform version to match the target version of Android Studio.
+
+![Example Android Studio About Dialog](img/android_studio_build.png){:width="600px"}
+
+Using Gradle as the basis for developing Android Studio plugins means the version of the IntelliJ IDEA IDE used to do the actual development does not have to match the targeted version of Android Studio.
```

Comment:
that is always true regardless of target IDE/version, we could move that paragraph to the Gradle topic instead